lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -303,11 +303,13 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 		}
 #endif
 
-		// Is the change a Transform3d change?
-		const TfToken nameToken = changedPath.GetNameToken();
-		if(nameToken == UsdGeomTokens->xformOpOrder || UsdGeomXformOp::IsXformOp(nameToken))
-		{
-			Ufe::Transform3d::notify(ufePath);
+		if (!InTransform3dChange::inTransform3dChange()) {
```

Comment:
Now account for a notification guard that I added in this branch, to prevent multiple changes to the data model (e.g. adding a transform op and setting its value, or adding a pivot op and its inverse) from notifying multiple times.